src/pkg/services/m365/api/pagers/pagers.go

```diff
@@ -40,9 +40,9 @@ type NextPager[T any] interface {
 	NextPage() (items []T, reset, done bool)
 }
 
-type nextPage[T any] struct {
-	items []T
-	reset bool
+type NextPage[T any] struct {
+	Items []T
+	Reset bool
```

Comment:
I only did some quick greps on the branch, but I don't think these fields need to be exported

Reply:
done; was autopiloting a bit.  thanks for the check.

Reply:
Please be more attentive going forward. No matter how the code came about, as the code author and author of the PR you're responsible for vouching for the general correctness and quality of the code. Expecting reviewers to check for simple quality things places a _huge_ burden in terms of review time and effort on them if you also want actionable and correct review comments. Just figuring out what could and couldn't be exported in these structs required me to either remember all the usages of these variables or clone the branch myself and search through the code using other tools

The bottom line is if the team at large keeps doing this either the review quality will go down (non-actionable or incorrect comments), the code quality will go down (more difficult to make changes going forward), or the reviewers are going to become overwhelmed